cmd/loop/loopout.go

```diff
@@ -235,6 +243,15 @@ func loopOut(ctx *cli.Context) error {
 		}
 	}
 
+	paymentTimeout := 0
+	if ctx.IsSet("payment_timeout") {
+		paymentTimeout = int(ctx.Duration("payment_timeout").Seconds())
```

Comment:
I propose to add a check that the value is divisible by a second. The current implementation treats `--payment_timeout=99.999s` as `--payment_timeout=99s` silently. I think it should crash instead, complaining that sub-second accuracy is not supported.